### PR TITLE
Fixing broken links in docs

### LINKED ISSUE #1227: [BUG] Broken link on docs.rs
## Bug Report
The link to the "zkVM tutorial" here is dead
image

### LINKED ISSUE #1388: [BUG] broken links on landing page for docs.rs zkvm crate
## Bug Report
The links to `receipt` on https://docs.rs/crate/risc0-zkvm/latest are broken. 

## Steps to Reproduce
1. Go to https://docs.rs/crate/risc0-zkvm/latest. Click the link for `receipt`. 

## Expected behavior
The link would get you to here: https://docs.rs/risc0-zkvm/latest/risc0_zkvm/struct.Receipt.html

## Your Environment
`docs.rs` on firefox

## Additional context
Notably, the link *does* work from https://docs.rs/risc0-zkvm/latest/risc0_zkvm/

### PR DESCRIPTION
**Pull request by flaub**
* Fixes #1227
* Fixes #1388

### CODE REVIEW

**Review by tzerrell — APPROVED** [review]
Nice!

File: risc0/zkvm/src/lib.rs [hunk 1]
```diff
@@ -12,11 +12,63 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#![doc = include_str!("../README.md")]
```
Comment (SchmErik) [inline risc0/zkvm/src/lib.rs:15]: I see what's going on here... A future improvement would be to figure out a way to strip off the `<!-- cargo-rdme ... -->`. I think this is good for the time-being though.

**Review by SchmErik — APPROVED** [review]

File: risc0/zkvm/src/lib.rs (hunk 1 above)
Comment (flaub) [reply risc0/zkvm/src/lib.rs:15]: It's kinda how `cargo-rdme` works, and I'm not sure it's an issue?

Comment (SchmErik) [reply risc0/zkvm/src/lib.rs:15]: Oh I had a misunderstanding! Please disregard this comment!

**Pull request merged**